documentation/src/main/asciidoc/userguide/chapters/jdbc/Database_Access.adoc

```diff
@@ -143,7 +143,7 @@ Additionally, this `ConnectionProvider` will pick up the following Hibernate-spe
 
 [IMPORTANT]
 ====
-To use this integration, the application must include the hibernate-agroal module jar (as well as its dependencies) on the classpath.
+To use the Agroal integration, the application must include the `hibernate-agroal` module jar (as well as its dependencies) on the classpath.
```

Comment:
Not related to this PR but don't you think we should reorder this list of pooling technologies with the most effective first (especially having c3p0 first sounds like a bad idea).

Reply:
c3p0 is definitely not a bad idea. It's quite a mature solution and running on hundreds of thousands of servers in production.

However, Proxool should be the last one mentioned. We could list Hikari, Vibur, and Agroal first, then c3p0 and Proxool.

Reply:
@vladmihalcea yes that's the idea: list first the ones we recommend.